### PR TITLE
ci: #1217 fix aws cp command to exclude sw.js and index.html for caching

### PR DESCRIPTION
**Pull request by tanphamhaiduong**
# Pull request checklist

## Does this close any currently open issues?

fixes: #1217 

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Deployment for development scripts to remove cache from sw.js and index.html

## What is the current behavior?

Issue Number: #1217 

## What is the new behavior?

- Copy script will exclude index.html and sw.js for caching from S3

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: scripts/release/release-dev.js [hunk 1]
```diff
@@ -21,19 +22,18 @@ const releaseDev = () => {
...
-      distPath,
-      `s3://${bucketName}`,
-      '--grants',
-      'read=uri=http://acs.amazonaws.com/groups/global/AllUsers',
-      '--recursive',
-      '--cache-control',
-      'max-age=31536000',
-    ])
+    const deleteResult = execSync(`aws s3 rm --recursive s3://${bucketName}`).toString()
+    console.log(deleteResult)
```
Comment (nphivu414) [inline scripts/release/release-dev.js:26]: Remove console.log

File: scripts/release/release-dev.js [hunk 2]
```diff
@@ -21,19 +22,18 @@ const releaseDev = () => {
...
+    // cp all assert with cache-control 365 days exclude sw.js and index.html
+    const copyWithCache = execSync(
+      `aws s3 cp ${distPath} s3://${bucketName} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --recursive --exclude "index.html" --exclude "sw.js" --exclude "config.json" --cache-control "max-age=31536000"`,
+    ).toString()
+    console.log(copyWithCache)
```
Comment (nphivu414) [inline scripts/release/release-dev.js:31]: Remove console.log

File: scripts/release/release-dev.js [hunk 3]
```diff
@@ -21,19 +22,18 @@ const releaseDev = () => {
...
+    // cp index.html and sw.js with no-cache control
+    const copyWithNoCache = execSync(
+      `aws s3 cp ${distPath} s3://${bucketName} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --recursive --exclude "*" --include "sw.js" --include "index.html" --include "config.json"`,
+    ).toString()
+    console.log(copyWithNoCache)
```
Comment (nphivu414) [inline scripts/release/release-dev.js:36]: Remove console.log

File: scripts/release/release-dev.js (hunk 2 above)
Comment (tanphamhaiduong) [reply scripts/release/release-dev.js:31]: This one will need to console to view the run result

File: scripts/release/release-dev.js (hunk 3 above)
Comment (tanphamhaiduong) [reply scripts/release/release-dev.js:36]: This one will need to console to view the result after run

**Pull request merged**